lib/websql/WebSQLDatabase.js

```diff
@@ -14,6 +14,22 @@ var COMMIT = [
   {sql: 'END;', args: []}
 ];
 
+var createInfoTable = {
+  sql: 'CREATE TABLE IF NOT EXISTS __WebKitDatabaseInfoTable__ ' +
+    '(key TEXT NOT NULL ON CONFLICT FAIL UNIQUE ON CONFLICT REPLACE,' +
```

Comment:
Why are there two `ON CONFLICT [...]` clauses here? I assume that the second one (`ON CONFLICT REPLACE`) wins?